samples/sensor/mcp9808/sample.yaml

```diff
@@ -6,3 +6,4 @@ tests:
     tags: sensors
     depends_on: i2c
     filter: dt_compat_enabled("microchip,mcp9808")
+    platform_allow: nucleo_l053r8
```

Comment:
not sure if this is strictly needed

Reply:
Not necessary, but I think good to add.

Addition was requested here: https://github.com/zephyrproject-rtos/zephyr/pull/31758#discussion_r566399165